osu.Game.Tests/Visual/Gameplay/TestScenePlayerLoader.cs

```diff
@@ -178,10 +183,13 @@ public void TestLoadContinuation()
 
             AddStep("load slow dummy beatmap", () =>
             {
+                prepareBeatmap();
                 LoadScreen(loader = new TestPlayerLoader(() => slowPlayer = new SlowLoadPlayer(false, false)));
```

Comment:
Maybe just instantiate it outside?
```
slowPlayer = new SlowLoadPlayer()
LoadScreen(loader = new TestPlayerLoader(() => slowPlayer);
```
My thinking is that the step could incorrectly refer to a previous (not-null) instance as it currently is.

(in both cases)

Reply:
Did consider that, but it's a sole test so that doesn't look possible.

Reply:
I think your way reads better, but it also potentially creates a weird timing scenario, since the schedule will happen far before the drawable is ready to process it, which could lead to subsequent assertion failures? Not sure..

Reply:
I'm confused, how is that not possible already?

Reply:
I guess it's much of a muchness, as the instance being instantiated doesn't signify load completion either.